src/SerilogTracing/ActivityListenerConfiguration.cs

```diff
@@ -79,10 +79,12 @@ public ActivityListenerConfiguration()
     /// The logger instance to emit traces through. Avoid using the shared <see cref="Log.Logger" /> as
     /// the value here. To emit traces through the shared static logger, call <see cref="TraceToSharedLogger" /> instead.
     /// </param>
+    /// <param name="ignoreLevelChanges">If <c langword="true"/>, the activity listener will assume all initial and
+    /// minimum levels are fixed at the time of activity listener creation. This may slightly improve tracing performance.</param>
     /// <returns>A handle that must be kept alive while tracing is required, and disposed afterwards.</returns>
-    public IDisposable TraceTo(ILogger logger)
+    public IDisposable TraceTo(ILogger logger, bool ignoreLevelChanges = false)
```

Comment:
Is this how we want to expose this setting?

Reply:
Thanks for the review. I tried `.Options.IgnoreLevelChanges()` first, which also felt wrong - top level properties in the configuration syntax are also prime real estate :-)

Another option, we could commit the internals now, but remove the parameter and spend some more time thinking about the API?

I'd love to just avoid it entirely, but haven't been able to come up with a satisfactory approach so far.

Reply:
I'd be ok to make it internal for a start and revisit in the future :+1: We may want to integrate it into the `InitialLevel` configuration somehow.